src/javascript/app/base/header.js

```diff
@@ -169,6 +170,15 @@ const Header = (() => {
                 on_mobile: true,
 
             },
+            ...(is_svg ? {
```

Comment:
Condition needs to be updated.
For logged in users:
- we need to display based on landing_companies, `dxtrade_gaming_company`

For logged out users:
- if it is not eu 
- if it is not an eu country